components/game/screen.js

```diff
@@ -25,6 +25,10 @@ class GameScreen {
     this.shiftSound = document.getElementById("sound-shift");
     this.clearLineSound = document.getElementById("sound-clear_line");
     this.gameOverSound = document.getElementById("sound-game_over");
+    this.background = {}
+    this.background.canvas = document.getElementById("background");
+    this.background.ctx = this.background.canvas.getContext("2d")
+    this.shouldUpdateStackedLayer = false;
```

Comment:
What is this block's purpose?
I couldn't find make this value false.

Reply:
Oh I see, It was mistaken.
The expected work is when the flag is true, this should new update render the stacked blocks.
Therefore, I've add clear block function which used clear marked blocks which is marked as previously rendered block.